bin/tronview

```diff
@@ -95,16 +96,24 @@ def view_job(args, job_id, client):
 
 
 def view_job_run(args, job_run_id, client):
-    actions = client.job_runs(job_run_id.url)
+    try:
+        actions = client.job_runs(job_run_id.url)
+    except RequestError as e:
```

Comment:
You could put this try/except in get_view_output, where the `obj_type_to_view_map` function is called on the client, to avoid repeating.